contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How Y Combinator is building the next generation of startups, with help from PostHog
+customer: Y Combinator
+logo: ../images/customers/ycombinator/yc_logo_b.png
+featuredImage: ../images/customers/ycombinator/featured.svg
+featuredCustomer: true
+industries:
+    - SaaS, Education
+users:
+    - Leadership
+    - Engineering
+    - Product
+toolsUsed:
+    - Cohorts
+    - Autocapture
+    - PostHog Cloud
+    - Insights
+---
+
+Y Combinator is the world’s top startup accelerator, helping to fund, train and guide the next generation of innovative businesses. Since 2005, thousands of successful companies have emerged from Y Combinator, including Dropbox, Airbnb, PagerDuty, Reddit, Amplitude …and PostHog. 
+
+“Getting into Y Combinator’s W20 batch was so important for us,” said James Hawkins, PostHog’s co-founder and CEO. “The programme helped us in so many ways, from advice on [Hacker News launch](https://news.ycombinator.com/item?id=22376732) to [helping us find our first five hires](/blog/posthog-first-five)...It’s incredibly satisfying that the team continues to use PostHog themselves!”
+
+“We use PostHog for our Startup School, for our public YC Startup Library and for a product called Co-Founder Matching,” said Cat Li, Product and Engineering Lead for Y Combinator Startup School. “PostHog is especially useful for Co-Founder Matching, which is like a dating app for co-founders.”
+
+### How Y Combinator gathers 30% more data with PostHog than Google Analytics
+
+PostHog isn’t the only tool in Y Combinator’s stack, of course. Teams across the organization can each have their own processes, tools and OKRs to guide their progress. Some teams at YCombinator use Segment or Amplitude — but for the flagship Startup School project, PostHog was a clear choice. 
+
+“We were very thoughtful about what we wanted to use with Startup School and Cofounder Matching,” said Cat. “We chose PostHog so we wouldn’t lose a bunch of the traffic and data. We found that with Google Analytics and other platforms we would lose up to 30% of our data from users who would not allow third-party tracking, or used adblockers. Our users are very tech-savvy…very likely to block third-party cookies.”
+
+“Because Startup School is very wide reaching, it really matters to us that we have an accurate  understanding of what's happening. That meant PostHog was the obvious choice.”
```

Comment:
```suggestion
“Because Startup School is very wide reaching, it really matters to us that we have an accurate understanding of what's happening. That meant PostHog was the obvious choice.”
```